app/voip/server/api/v1/Rooms.ts

```diff
@@ -0,0 +1,86 @@
+import { Meteor } from 'meteor/meteor';
+import { Match, check } from 'meteor/check';
+import { Random } from 'meteor/random';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+
+import { API } from '../../../../api/server';
+import { VoipRoom } from '../../../../models/server/raw';
+import { Voip } from '../../lib/Voip';
+import { OmnichannelSourceType } from '../../../../../definition/IRoom';
+
+API.v1.addRoute('voip/room', {
```

Comment:
Again, if the voip room is a new type, couldn't we use our current endpoints and only filter by type 'v' ?

Reply:
I am not sure if this comment is after our discussions. Nevertheless I will add some context for documenting the discussion.We have decided to create a new API/Wrapper collection for Voip/Voice rooms. The reason for not using the current endpoints is that we will need a different behaviour for the voip rooms. e.g We do not need queues. We may need subscription. The routing is done via asterisk. So to avoid overloading the existing code (Which may result in regression), we have decided to use new endpoint/collection wrappers for Voip.